tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
```

Comment:
I don't like squashing, now we have only one commit and don't see the history, it's harder to see it and play with it.

so, guys, let say I want to make `@Foo` ruleset, and then `@Bar` ruleset, which is the same as `@Foo` but without one rule (and having some extra rule). How could I define `@Bar` set?
```
'@Bar' => {
    '@Foo` => true,
    'add_this_rule` => true,
    'remove_this_rule` => ???
},
```
?